pkg/api/controllers/script_controller_test.go

```diff
@@ -29,44 +30,50 @@ func TestPostScript(t *testing.T) {
 		{
 			name: "nominal",
 			script: core.Script{
-				Plain: `
-				send [COIN 100] (
-				  source = @world
-				  destination = @centralbank
-				)
-				send [COIN 100] (
-				  source = @centralbank
-				  destination = @users:001
-				)`,
+				ScriptCore: core.ScriptCore{
+					Plain: `
+					send [COIN 100] (
+					  source = @world
+					  destination = @centralbank
+					)
+					send [COIN 100] (
+					  source = @centralbank
+					  destination = @users:001
+					)`,
+				},
 			},
 		},
 		{
-			name: "failure with insufficient funcs",
+			name: "failure with insufficient funds",
 			script: core.Script{
-				Plain: `
-				send [COIN 100] (
-				  source = @centralbank
-				  destination = @users:001
-				)`,
+				ScriptCore: core.ScriptCore{
+					Plain: `
+					send [COIN 100] (
+					  source = @centralbank
+					  destination = @users:001
+					)`,
+				},
 			},
 			expectedResponse: controllers.ScriptResponse{
 				ErrorResponse: sharedapi.ErrorResponse{
-					ErrorCode:    ledger.ScriptErrorInsufficientFund,
```

Comment:
Don't move all errors at api level.
Errors coming from the ledger package must stay on the ledger package.